mmpose/utils/config_utils.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmpose.utils.typing import ConfigDict
+
+
+def convert_mmdet_test_pipeline(cfg: ConfigDict) -> ConfigDict:
```

Comment:
What about naming the function 'port_mmdet_pipeline' or 'adapt_mmdet_pipeline'?

Reply:
Good idea. I think 'adapt_mmdet_pipeline' is preferable